2022-09-22 16:41:10.498 +02:00 [INF] Application is RUNNING..... follow her x)
2022-09-22 16:41:32.528 +02:00 [INF] Web Api is runninggggg...
2022-09-22 16:45:48.664 +02:00 [INF] Application is RUNNING..... follow her x)
2022-09-22 16:46:31.666 +02:00 [INF] Application is RUNNING..... follow her x)
2022-09-22 16:47:14.068 +02:00 [INF] Application is RUNNING..... follow her x)
2022-09-23 09:50:07.562 +02:00 [INF] Application is RUNNING..... follow her x)
2022-09-23 09:50:28.822 +02:00 [WRN] Get item with id '3' no found
2022-09-23 09:50:29.871 +02:00 [ERR] An unhandled exception has occurred while executing the request.
System.Collections.Generic.KeyNotFoundException: User not found
   at WebDemo.Api.Services.UserService.FindUserByIdAsync(Int32 id) in C:\Users\youba.imakhlaf\source\repos\WebDemo\WebDemo.Api\Services\UserService.cs:line 101
   at WebDemo.Api.Services.UserService.GetUserAsync(Int32 id) in C:\Users\youba.imakhlaf\source\repos\WebDemo\WebDemo.Api\Services\UserService.cs:line 61
   at WebDemo.Api.Controllers.V1.UsersController.Get(Int32 id) in C:\Users\youba.imakhlaf\source\repos\WebDemo\WebDemo.Api\Controllers\V1\UsersController.cs:line 35
   at lambda_method12(Closure , Object )
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.AwaitableObjectResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|25_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Serilog.AspNetCore.RequestLoggingMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.Invoke(HttpContext context)
2022-09-23 10:17:41.007 +02:00 [INF] Application is RUNNING..... follow her x)
2022-09-23 10:18:05.766 +02:00 [WRN] Get item with id '4' no found
2022-09-23 10:18:15.000 +02:00 [WRN] Get item with id '3' no found
2022-09-23 10:18:15.223 +02:00 [ERR] An unhandled exception has occurred while executing the request.
System.Collections.Generic.KeyNotFoundException: User not found
   at WebDemo.Api.Services.UserService.FindUserByIdAsync(Int32 id) in C:\Users\youba.imakhlaf\source\repos\WebDemo\WebDemo.Api\Services\UserService.cs:line 101
2022-11-04 14:41:58.221 +01:00 [INF] Timed Hosted Service is working. Count: 2
2022-11-04 14:42:03.220 +01:00 [INF] Timed Hosted Service is working. Count: 3
2022-11-04 14:42:08.223 +01:00 [INF] Timed Hosted Service is working. Count: 4
2022-11-04 14:42:13.235 +01:00 [INF] Timed Hosted Service is working. Count: 5
2022-11-04 14:42:18.231 +01:00 [INF] Timed Hosted Service is working. Count: 6
2022-11-04 14:42:23.235 +01:00 [INF] Timed Hosted Service is working. Count: 7
2022-11-04 15:25:35.769 +01:00 [INF] Worker running at: "2022-11-04T15:25:35.7660590+01:00"
2022-11-04 15:35:51.679 +01:00 [INF] BgUserService running at: "2022-11-04T15:35:51.6759458+01:00"

